### PR TITLE
Some Dockerfile improvements

### PR DESCRIPTION
**Pull request by frauhottelmann**
I improved the Dockerfile a bit. Some cosmetics, some Dockerfile best practices. I added net-tools for the Server-Status page. I also added pear Mail and Maile-Mime as suggested in the requirements. I deleted the explicit Ubuntu mirrors to let Ubuntu choose it's own download location.

Maybe it's also time to switch to Ubtunu 18.04? My own Daloradius Docker image works fine with it.

The Dockerfile does build and start fine.

### CODE REVIEW

**Comment by frauhottelmann** [comment]
Also the mysql password shouldn't be empty, especially since the MySQL-port is being exposed...

File: Dockerfile-freeradius [hunk 1]
```diff
@@ -3,7 +3,7 @@
 #
 # Build image:
 # 1. git pull git@github.com:lirantal/daloradius.git
-# 2. docker build . -t lirantal/daloradius
+# 2. docker build . -t lirantal/daloradius -f Dockerfile-freeradius
```
Comment (lirantal) [inline Dockerfile-freeradius]: the `.` already specifies the dockerfile already, no? isn't that making the `-f` at the end extranous? I assume it is one or the other.

Comment (frauhottelmann) [reply Dockerfile-freeradius]: The . would build the Dockerfile and not the Dockerfile-freeradius. That is why I added it.

**Comment by lirantal** [comment]
@frauhottelmann good stuff and great comments too, thank you.
I definitely agree about MySQL. Is that a default with MySQL installations these days? I haven't done one for ages. I ask, because if we make a default MySQL use a password then we also need to change the default password in `daloradius.conf.php` and if both are anyway public then it is already public knowledge and loses the meaning of it.

I'm happy to merge once you confirm.
Did you also mean to make a change to the default Dockerfile?

File: Dockerfile-freeradius (hunk 1 above)
Comment (lirantal) [reply Dockerfile-freeradius]: What I meant is that I think the correct syntax is this, no?

```
docker build -t lirantal/daloradius -f Dockerfile-freeradius
```

**Comment by frauhottelmann** [comment]
We could leave it blank and assume that users would change it via ENV. Or add a comment there.
I can fix up the other Dockerfile as well, but I haven't looked at it yet.

File: Dockerfile-freeradius (hunk 1 above)
Comment (frauhottelmann) [reply Dockerfile-freeradius]: Ah, missed the . 
You are correct.

**Comment by frauhottelmann** [comment]
Or we don't expose 3306 by default. If someone needs access to the MySQL server they can expose it at run time.

**Comment by lirantal** [comment]
Uhmmm, yeah we don't need to expose it since this Dockerfile bundles everything so all the networking is done internally on the container using localhost, right? If so, sounds good with me.

**Comment by frauhottelmann** [comment]
> Uhmmm, yeah we don't need to expose it since this Dockerfile bundles everything so all the networking is done internally on the container using localhost, right? If so, sounds good with me.

Yes, internally it's no problem. `EXPOSE 3306` makes MySQL accessible outside the container.

**Comment by lirantal** [comment]
Alright, so whenever you push the changes just ping me and I'll merge it in.

**Comment by lirantal** [comment]
Sounds good, thank you! ❤️

**Pull request merged**